### DISCUSSION TITLE
Tool to take difference between last value and preceding value

### DISCUSSION THREAD

**Discussion by jppo** [grafana/grafana#64341] [General]
Hello,

As many tools return cumulative indicators it would be interesting to be able to use the difference between the values of N and N-1 to get more "informative" data.

Regards

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This feature request is fairly vague, and I'm wondering if you can provide more context in an issue ticket. Feel free to make an [issue](https://github.com/grafana/grafana/issues) about what it is youre looking for.
